transactions/add-account/2024/aug-27/create_account.cdc

```diff
@@ -0,0 +1,38 @@
+import FlowToken from 0x1654653399040a61
```

Comment:
@sideninja can you please review this transaction?

Reply:
Why does this transaction need to be executed by the service account?

Reply:
tbh it doesn't has to be. We thought it will be good for an audit purpose where one can clearly see the gateway account being created by the service account.